### DISCUSSION TITLE
Changing Language and Theme for the UI

### DISCUSSION THREAD

**Discussion by vraixen** [sileshn/LinuxmintWSL#5] [Q&A] [answered]
How can i change language? also i would like to change the themes of the UI (GTK?)

**Comment by sileshn** [comment]
If you want another language, you can manually change the text in bash_profile and compile again. I am not aware of another way to do it.

What do you mean by UI? If you are referring to the colored text displayed during installation, its done using figlet and lolcat. You can change the font and colors in the bash_profile file to your liking. 

You will have to compile after making changes.

**Reply by vraixen** [reply, marked as answer]
for example if you install a program that has a GUI you can open it.. but that program will have the default white theme. Do you know if there's a way to change it?
![immagine](https://github.com/sileshn/LinuxmintWSL2/assets/58710194/df0716ef-d697-48ad-8503-afb0ea5760a9)

**Comment by sileshn** [comment]
https://github.com/microsoft/wslg/issues/251
